spec/index.bs

```diff
@@ -7281,6 +7281,23 @@ Issue: Because timestamp query provides high-resolution GPU timestamp, we need t
 
 # Canvas Rendering &amp; Swap Chains # {#canvas-rendering}
 
+## HTMLCanvasElement.getContext ## {#canvas-getcontext}
+
+A {{GPUCanvasContext}} object can be obtained via the `getContext` method of a {{HTMLCanvasElement}} instance by
+passing the string literal `'gpupresent'` as its `contextType` argument.
+
+<div class="example">
+    Get a {{GPUCanvasContext}} from an offscreen {{HTMLCanvasElement}}:
+    ```ts
```

Comment:
As tempting as it is to use TypeScript, I think we should stick to plain JS in the spec 🙂

Reply:
Yeah... I had a dream :)
Fixed.